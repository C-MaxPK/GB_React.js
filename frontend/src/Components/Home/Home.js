import { PropTypes } from 'prop-types';
import './home.css';

const Home = ({ firstProp }) => {
    
    return (
        <h2 className="firstStyle">
            { firstProp }
        </h2>
    )
};

Home.propTypes = {
    firstProp: PropTypes.string.isRequired
};

export default Home;
