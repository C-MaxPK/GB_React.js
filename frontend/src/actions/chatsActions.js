import { SET_CHATS } from "../store/types/chatsType";

export const addChatItem = (obj) => (dispatch) => {
    fetch('http://localhost:3001/chats', {
        method: 'POST',
        headers: {
            "Content-Type": "application/json"
        },
        body: JSON.stringify(obj)
    })
        .then(response => response.json())
        .then(newData => dispatch(setChats(newData)));
};

export const deleteChatItem = (id) => (dispatch) => {
    fetch(`http://localhost:3001/chats?id=${id}`, {
        method: 'DELETE'
    })
        .then(response => response.json())
        .then(newData => dispatch(setChats(newData)));
};

export const setChats = (obj) => ({
    type: SET_CHATS,
    payload: obj
})

export const fetchChats = () => (dispatch) => {
    fetch('http://localhost:3001/chats')
        .then(response => response.json())
        .then(data => dispatch(setChats(data)));
}
