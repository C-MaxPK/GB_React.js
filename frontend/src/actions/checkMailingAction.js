import { CHANGE_CHECK_MAILING } from "../store/types/checkMailingType";

export const changeCheckMailing = () => ({
    type: CHANGE_CHECK_MAILING
});
