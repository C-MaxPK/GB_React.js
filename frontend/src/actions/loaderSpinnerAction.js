import { LOADING_ON } from "../store/types/loaderSpinnerType";

export const setLoaderSpinner = (value) => ({
    type: LOADING_ON,
    payload: value
});
