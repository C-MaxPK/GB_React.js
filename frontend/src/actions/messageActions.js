import { SET_MESSAGES } from "../store/types/messageType";

export const addMessageItem = (obj) => (dispatch) => {
    fetch('http://localhost:3001/messages', {
        method: 'POST',
        headers: {
            "Content-Type": "application/json"
        },
        body: JSON.stringify(obj)
    })
        .then(response => response.json())
        .then(newData => dispatch(setMessages(newData)));
};

export const setMessages = (obj) => ({
    type: SET_MESSAGES,
    payload: obj
})

export const showAutoAnswer = (chatId) => (dispatch, getState) => {
    setTimeout(() => {
        console.log(`Я робот-автоответчик, а вы: ${getState().messageReducer[chatId][getState().messageReducer[chatId].length - 1].author} - кожаный человек`);
    }, 1500);
};

export const fetchMessages = () => (dispatch) => {
    fetch('http://localhost:3001/messages')
        .then(response => response.json())
        .then(data => dispatch(setMessages(data)));
}

export const deleteMessagesChat = (id) => (dispatch) => {
    fetch(`http://localhost:3001/messages?id=${id}`, {
        method: 'DELETE'
    })
        .then(response => response.json())
        .then(newData => dispatch(setMessages(newData)));
};
