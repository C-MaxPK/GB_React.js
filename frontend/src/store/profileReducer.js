import { CHANGE_CHECK_MAILING } from "./types/checkMailingType";

const initialState = {
    checkMailing: false
};

const profileReducer = (state = initialState, {type}) => {
    switch (type) {
        case CHANGE_CHECK_MAILING:
            return {...state, checkMailing: !state.checkMailing};
        default:
            return state;
    }
};

export default profileReducer;
