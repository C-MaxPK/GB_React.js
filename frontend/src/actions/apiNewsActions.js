import { ADD_API_NEWS } from "../store/types/apiNewsType";
import { setError } from "./errorAction";
import { setLoaderSpinner } from "./loaderSpinnerAction";

export const addApiNews = (obj) => ({
    type: ADD_API_NEWS,
    payload: obj
});

export const getApiNews = (theme) => (dispatch) => {
    dispatch(setLoaderSpinner(true));
    fetch(`https://inshortsapi.vercel.app/news?category=${theme}`)
        .then(response => response.json())
        .then(data => {
            dispatch(addApiNews(data.data));
            dispatch(setLoaderSpinner(false));
        })
        .catch(err => {
            dispatch(addApiNews([]));
            dispatch(setError(err));
            dispatch(setLoaderSpinner(false));
        });
};
