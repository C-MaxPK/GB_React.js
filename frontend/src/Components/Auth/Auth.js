import { useEffect, useState } from "react";
import { Box, Button, Modal, TextField } from '@material-ui/core';
import { useHistory } from 'react-router-dom';
import { useDispatch, useSelector } from "react-redux";
import { fetchUser } from '../../actions/userActions';
import './auth.css';

const Auth = () => {
    const [email, setEmail] = useState('');
    const [password, setPassword] = useState('');
    const [errorFlg, setErrorFlg] = useState(false);
    const dispatch = useDispatch();
    const history = useHistory();
    const { error, user } = useSelector(state => state.userReducer);

    const style = {
        position: 'absolute',
        top: '50%',
        left: '50%',
        transform: 'translate(-50%, -50%)',
        width: 600,
        bgcolor: 'background.paper',
        boxShadow: 24,
        p: 2,
    };

    useEffect(() => {
        if (error) {
            setEmail('');
            setPassword('');
            setErrorFlg((prev) => !prev);
        }
    }, [error]);

    useEffect(() => {
        if (user) {
            history.push('/home');
        }
    }, [user]);
    
    const submitHandler = (e) => {
        e.preventDefault();
        // dispatch(userLogin({ email, password }));
        dispatch(fetchUser({ email, password }));
        setErrorFlg(false);
    };

    return (
        <Modal open={true} >
            <Box sx={style}>
                <form className='formAuth' onSubmit={submitHandler}>
                    <TextField value={email} onChange={e => setEmail(e.target.value)} required id="outlined-required" label="E-Mail" type="email" />
                    <TextField value={password} onChange={e => setPassword(e.target.value)} id="outlined-password-input" label="Password" type="password" autoComplete="current-password" />
                    <Button type='submit' variant="outlined">LogIn</Button>
                </form>
                {errorFlg && (<div>ERROR AUTH</div>)}
            </Box>
        </Modal>
    )
};

export default Auth;
