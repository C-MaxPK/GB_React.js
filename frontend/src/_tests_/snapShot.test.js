import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import Auth from '../Components/Auth/Auth';
import Home from '../Components/Home/Home';
import { store } from '../store';
import { changeCheckMailing } from '../actions/checkMailingAction';
import profileReducer, { initialState } from '../store/userReducer';

describe('test reducers', () => {
    it('test profileReducer', () => {
        const reducer = profileReducer(initialState, changeCheckMailing());
        expect(reducer).toMatchSnapshot();
    });
});

describe('test components', () => {
    it('test Home component', () => {
        const component = render(
            <Provider store={store}>
                <Home />
            </Provider>
        );
         expect(component).toMatchSnapshot();
    });

    it('test Auth component', () => {
        const component = render(
            <Provider store={store}>
                <Auth />
            </Provider>
        );
         expect(component).toMatchSnapshot();
    });
});
