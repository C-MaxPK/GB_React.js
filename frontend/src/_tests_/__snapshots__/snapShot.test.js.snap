// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test components test Auth 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiModal-root css-79ws1d-MuiModal-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-i9fmh8-MuiBackdrop-root-MuiModal-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiBox-root css-1im9xjz"
        tabindex="-1"
      >
        <form
          class="formAuth"
        >
          <div
            class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for="outlined-required"
              id="outlined-required-label"
            >
              E-Mail
              <span
                aria-hidden="true"
                class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                id="outlined-required"
                required=""
                type="email"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1o2jng6-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-2s9zix"
                >
                  <span>
                    E-Mail
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for="outlined-password-input"
              id="outlined-password-input-label"
            >
              Password
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                autocomplete="current-password"
                class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                id="outlined-password-input"
                type="password"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1o2jng6-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-2s9zix"
                >
                  <span>
                    Password
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            LogIn
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </form>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`test components test Auth component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiModal-root css-79ws1d-MuiModal-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-i9fmh8-MuiBackdrop-root-MuiModal-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiBox-root css-1im9xjz"
        tabindex="-1"
      >
        <form
          class="formAuth"
        >
          <div
            class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary Mui-required css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for="outlined-required"
              id="outlined-required-label"
            >
              E-Mail
              <span
                aria-hidden="true"
                class="MuiInputLabel-asterisk MuiFormLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
              >
                 
                *
              </span>
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                id="outlined-required"
                required=""
                type="email"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1o2jng6-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-2s9zix"
                >
                  <span>
                    E-Mail
                     *
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <div
            class="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
          >
            <label
              class="MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined MuiFormLabel-root MuiFormLabel-colorPrimary css-14s5rfu-MuiFormLabel-root-MuiInputLabel-root"
              data-shrink="false"
              for="outlined-password-input"
              id="outlined-password-input-label"
            >
              Password
            </label>
            <div
              class="MuiOutlinedInput-root MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
            >
              <input
                aria-invalid="false"
                autocomplete="current-password"
                class="MuiOutlinedInput-input MuiInputBase-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                id="outlined-password-input"
                type="password"
                value=""
              />
              <fieldset
                aria-hidden="true"
                class="MuiOutlinedInput-notchedOutline css-1o2jng6-MuiOutlinedInput-notchedOutline"
              >
                <legend
                  class="css-2s9zix"
                >
                  <span>
                    Password
                  </span>
                </legend>
              </fieldset>
            </div>
          </div>
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="submit"
          >
            LogIn
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </form>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`test components test Home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h2
        class="firstStyle"
      />
    </div>
  </body>,
  "container": <div>
    <h2
      class="firstStyle"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`test components test Home component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h2
        class="firstStyle"
      />
    </div>
  </body>,
  "container": <div>
    <h2
      class="firstStyle"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`test reducers test profileReducer 1`] = `
Object {
  "error": null,
  "user": null,
}
`;
